components/form/index.en-US.md

```diff
@@ -86,6 +87,7 @@ Form field component for data bidirectional binding, validation, layout, and so
 | labelCol | The layout of label. You can set `span` `offset` to something like `{span: 3, offset: 12}` or `sm: {span: 3, offset: 12}` same as with `<Col>`. You can set `labelCol` on Form. If both exists, use Item first | [object](/components/grid/#Col) | - |  |
 | name | Field name, support array | [NamePath](#NamePath) | - |  |
 | normalize | Normalize value from component value before passing to Form instance | (value, prevValue, prevValues) => any | - |  |
+| preserve | Keep field value even when field removed | boolean | true | 4.4.0 |
```

Comment:
Why `preserve` default to `true`?  `false` it's more reasonable for daily business IMO. Performance or any other reasons?
@zombieJ

Reply:
`preserve` makes user  easy to keep origin field data when switch makes field removed. You can set `preserve` on Form level to change it.

Reply:
Thx your reply. But according to https://github.com/ant-design/ant-design/issues/21740, we often use Modal `destroyOnClose` with Form `preserve=false`. And In [field-form](https://github.com/react-component/field-form#form), `preserve=false` is default choise, which makes me confused